openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-autonomic/src/tools/Autonomic/Tabs/Triggers/TriggerOperand.vue

```diff
@@ -205,29 +219,47 @@ export default {
     },
     limitTypes: function () {
       return [
-        { text: '', value: '' },
-        { text: 'LOW', value: '_LOW' },
-        { text: 'HIGH', value: '_HIGH' },
+        { text: 'LOW', value: 'LOW' },
+        { text: 'HIGH', value: 'HIGH' },
       ]
     },
     operandTypes: function () {
-      switch (this.kind) {
-        case 'FLOAT':
-          return ['ITEM', 'FLOAT']
-        case 'STRING':
-          return ['ITEM', 'STRING']
-        case 'ITEM':
-          return ['ITEM', 'FLOAT', 'STRING', 'LIMIT']
-        case 'TRIGGER':
-          return ['TRIGGER']
-        default:
-          return ['ITEM', 'FLOAT', 'STRING', 'LIMIT', 'TRIGGER']
+      if (this.order === 'left') {
+        return [
+          { text: 'Telemetry Item', value: 'ITEM' },
+          { text: 'Existing Trigger', value: 'TRIGGER' },
+        ]
```

Comment:
I'm limiting the left operand to ITEM or TRIGGER. I think it just makes more sense and we have both < and > so you can achieve any comparison.